pkg/visor/peering_daemon.go

```diff
@@ -0,0 +1,109 @@
+package visor
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net"
+	"net/rpc"
+	"os"
+	"os/exec"
+	"sync"
+	"time"
+
+	"github.com/SkycoinProject/dmsg/cipher"
+	skycoin_cipher "github.com/SkycoinProject/skycoin/src/cipher"
+	"github.com/SkycoinProject/skycoin/src/util/logging"
+	spd "github.com/SkycoinProject/skywire-peering-daemon/pkg/daemon"
+	"github.com/rjeczalik/notify"
+
+	"github.com/SkycoinProject/skywire-mainnet/pkg/snet"
+)
+
+var (
+	rpcAddr         = "localhost:3435"
+	logger          = logging.MustGetLogger("SPD")
+	rpcDialTimeout  = time.Duration(5 * time.Second)
+	rpcConnDuration = time.Duration(60 * time.Second)
+	spdMu           sync.Mutex
+)
+
+func execute(cmd *exec.Cmd) error {
+	cmd.Stdout = os.Stdout
+	if err := cmd.Start(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func client() (RPCClient, error) {
```

Comment:
How is this function related to the Peering Daemon?

Reply:
When a visor receives a packet from a peering daemon, it's supposed to automatically establish an `stcp` transport to a visor whose public key is contained in the (received) packet. 

The function doesn't in any way relate to the peering daemon. There's a similar function in `cmd/skywire-cli/commands/visor/root.go` which returns a RPC client needed to establish transport (dmsg/stcp).  Thinking of moving this function to `pkg/httputil` package, so it can be used in the peering_daemon.go file and also by `skywire-cli`

Reply:
@Kifen Thanks for the explanation and that does make more sense.

However, just a few notes:
1. `httputil` package already exists under `dmsg/httputil`. Remove `httputil` as it is a duplicate.
2. I feel like this function should be under the same package, just a different file. Consider putting this in `rpc_client.go`.

Reply:
For point 2, looking at the function again, already `NewRPCClient` exist in `rpc_client.go` and it returns an `rpcClient` required for making rpc calls. What the function `client()` does in peering_daemon.go is create a `net.Conn` type which is then used in the call to `NewRPCClient`.